docs/examples/general/expressions/expr_examples.ipynb

```diff
@@ -11,11 +11,26 @@
     "## Supported Operators\n",
```

Comment:
<p>I am not sure about usage of <em>susceptible</em> in this context.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4568/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>